lnbits/extensions/gerty/helpers.py

```diff
@@ -765,17 +776,31 @@ async def get_onchain_dashboard(gerty):
     areas = []
     if isinstance(gerty.mempool_endpoint, str):
         async with httpx.AsyncClient() as client:
+            text = []
```

Comment:
- can this new logic be one level up? Outside of:
```py
async with httpx.AsyncClient() as client:
```

Reply:
Thanks for the review Vlad. I'm not sure what you mean by this comment? Could you elaborate please and I'll address?

Reply:
hmm, I see now that `client` variable is not used.
So the bleow line can be removed entirely (and also reduce one level of nesting):
```py
async with httpx.AsyncClient() as client:
```

- same applies for the `client` variable in `get_onchain_stat`

Reply:
Done